scripts/update-go-pbs.sh

```diff
@@ -22,6 +22,20 @@ for ((i = 0; i < arraylength; i++)); do
     cp -R -L "${file_list[i]}" "$destination"
 done
 
+file_list=()
+while IFS= read -d $'\0' -r file; do
+    file_list=("${file_list[@]}" "$file")
+done < <($findutil -L "$(bazel info bazel-bin)"/proto/eth -type f -regextype sed -regex ".*pb\.\(gw\.\)\?go$" -print0)
+
+arraylength=${#file_list[@]}
+ethereumapis="prysmaticlabs/ethereumapis/"
```

Comment:
These are in a separate package so we need to add the this logic to copy successfully